docs/ui/components/ContentSpotlight/index.tsx

```diff
@@ -0,0 +1,124 @@
+  loop?: boolean;
+  style?: CSSProperties;
+  containerClassName?: string;
+};
+
+export function ContentSpotlight({
+  alt,
+  src,
+  url,
+  file,
+  caption,
+  controls,
+  spaceAfter,
+  loop = true,
+  style,
+  containerClassName,
+}: ContentSpotlightProps) {
+  const isYouTubeDomain = (url?: string) => {
+    return url ? YOUTUBE_DOMAINS.some(domain => url.includes(domain)) : false;
+  };
+
+  const getInitialMarginBottom = (spaceAfter: ContentSpotlightProps['spaceAfter']) => {
+    if (typeof spaceAfter === 'undefined') {
+      return 'mb-2';
+    } else if (typeof spaceAfter === 'number') {
+      return `mb-${spaceAfter}`;
+    } else if (spaceAfter) {
+      return 'mb-12';
+    }
+    return 'mb-0';
+  };
+
+  const [hover, setHover] = useState(false);
+  const [forceShowControls, setForceShowControls] = useState(isYouTubeDomain(url));
+  const isVideo = !!(url || file);
+
+  return (
+    <figure
+      className={mergeClasses(
+        'text-center py-2.5 my-5 rounded-lg',
+        containerClassName,
+        !isVideo && 'bg-subtle'
+      )}
+      onClick={() => {
+        if (typeof controls === 'undefined' && !forceShowControls) {
+          setForceShowControls(true);
+        }
+      }}
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      style={hover ? { cursor: 'pointer' } : undefined}>
+      {src ? (
+        <LightboxImage
+          src={src}
+          alt={alt}
+          style={style}
+          className="inline rounded-md transition-opacity duration-default ease-in-out hover:opacity-80"
+        />
+      ) : isVideo ? (
+        <VisibilitySensor partialVisibility>
+          {({ isVisible }: { isVisible: boolean }) => (
+            <div className={getInitialMarginBottom(spaceAfter)}>
+              <div className="relative w-full h-[400px] bg-palette-black rounded-lg overflow-hidden">
+                <ReactPlayer
+                  url={isVisible ? url || `/static/videos/${file}` : undefined}
+                  className="react-player"
+                  width={PLAYER_WIDTH}
+                  height={PLAYER_HEIGHT}
+                  style={{
+                    outline: 'none',
+                    backgroundColor: '#000',
+                  }}
+                  muted
+                  playing={isVisible && !!file}
+                  controls={typeof controls === 'undefined' ? forceShowControls : controls}
+                  playsinline
+                  loop={loop}
+                />
+                <div
+                  className={`pointer-events-none absolute inset-0 transition-opacity duration-500 ${
+                    isVisible ? 'opacity-0' : 'opacity-70'
+                  } md:hidden`}
+                />
+              </div>
+            </div>
+          )}
+        </VisibilitySensor>
+      ) : null}
+      {caption && (
+        <figcaption
+          className={`mt-[14px] text-secondary text-center text-xs px-8 py-2 ${
```

Comment:
```suggestion
          className={`mt-3.5 text-secondary text-center text-xs px-8 py-2 ${
```

Nit, half unit classes exist up to `4`: https://tailwindcss.com/docs/margin